cra-network-aware-data-fetching/src/components/SearchBar/index.js

```diff
@@ -0,0 +1,32 @@
+
+import React, { useState } from 'react';
+
+import './search-bar.css';
+
+const SearchBar = ({ search }) => {
```

Comment:
Do we currently throttle how often we query the API? We probably want to avoid doing this onChange for every keystroke, right?

Reply:
Now, we just call API once clicking on search icon, not calling API in every onChange event.
In every onChange event, we just update the input value.

Reply:
cross-reference: https://www.peterbe.com/plog/how-to-throttle-and-debounce-an-autocomplete-input-in-react

Reply:
Thanks, let me check and update the codes.

Reply:
Ah, okay - so testing out https://anton-karlovskiy-network-aware-data-fetching-for-glitch.glitch.me/ I see that you need to tap on the magnifying glass in order to search (vs it searching on key-press). Rather than change this, maybe we can add the ability to search if you hit the ENTER key too?

Reply:
No problem. :)

Reply:
I added Enter key feature. You can check it on glitch version as I updated it as well.